src/symbolize/gimli/parse_running_mmaps_unix.rs

```diff
@@ -44,13 +44,14 @@ pub(super) struct MapsEntry {
 }
 
 pub(super) fn parse_maps() -> Result<Vec<MapsEntry>, &'static str> {
-    let mut v = Vec::new();
     let mut proc_self_maps =
         File::open("/proc/self/maps").map_err(|_| "Couldn't open /proc/self/maps")?;
     let mut buf = String::new();
     let _bytes_read = proc_self_maps
         .read_to_string(&mut buf)
         .map_err(|_| "Couldn't read /proc/self/maps")?;
+
+    let mut v = Vec::new();
```

Comment:
kinda insane that this matters that much

Reply:
it's only a tiny improvement, so _that much_ is a bit of an overstatement :). actually i don't have numbers on this one, but I did see the call to `drop_in_place` before but not after.